app/org/zalando/hutmann/authentication/User.scala

```diff
@@ -43,11 +43,11 @@ object User {
   * Intermediate user that will only be used internally. Do not use this one directly, use App2AppUser instead.
   */
 private final case class IntermediateUser(
-  accessToken: String,
-  scope:       Set[String],
-  realm:       String,
-  tokenType:   String,
-  expiresIn:   Int
+    accessToken: String,
```

Comment:
I could imagine the formatting isn't always consistent and in this case later it is padded with two spaces. Do you see specific needs for formatting it differently?

Reply:
the reason of new formatting is the following change:
<img width="562" alt="screenshot 2019-02-13 at 14 08 33" src="https://user-images.githubusercontent.com/2256720/52713912-dfee9480-2f98-11e9-91c7-65effb7258f2.png">

With new scalariform version `doubleIndentClassDeclaration` got deprecated and we need to use `doubleIndentConstructorArguments` instead. `doubleIndentConstructorArguments: true` is [recommended by the Scala Style Guide](https://docs.scala-lang.org/style/declarations.html#classes), so the result should now be:
```
class Person(
    name: String,
    age: Int,
    astrologicalSign: String)
  extends Entity
  with Logging {
}
```

Before it was:
```
class Person(
  name: String,
  age: Int,
  astrologicalSign: String)
    extends Entity
    with Logging {
}
```

Reply:
Ah, so you run the formatter? If so it's great. Keep it then

Reply:
I only changed scalariform config, formatting is done on sbt build